controllers/keda/scaledobject_finalizer.go

```diff
@@ -78,10 +78,10 @@ func (r *ScaledObjectReconciler) finalizeScaledObject(ctx context.Context, logge
 			logger.Error(err, "Failed to update ScaledObject after removing a finalizer", "finalizer", scaledObjectFinalizer)
 			return err
 		}
```

Comment:
Don't include not neccessary changes in the PR, these empty lines are not doing anything